operators/go.mod

```diff
@@ -3,24 +3,33 @@ module github.com/netgroup-polito/CrownLabs/operators
 go 1.16
 
 require (
+	cloud.google.com/go v0.99.0 // indirect
```

Comment:
Changes to go-mod/go.sum shall be reverted, unless necessary.

Reply:
These are from the linter, can we leave them? @giorio94

Reply:
No, the PR should not include leftovers which are not required for its functioning.

Reply:
Resolved in [71caff6](https://github.com/netgroup-polito/CrownLabs/pull/821/commits/71caff6348e54c57bdf9d0e260066ad59ad68563)